test/built-ins/JSON/parse/reviver-call-err.js

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2016 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-internalizejsonproperty
+es6id: 24.3.1.1
+description: >
+  Abrupt completion when reviver function returns an abrupt completion
+info: |
+  7. If IsCallable(reviver) is true, then
```

Comment:
Would you mind putting something along the lines of:
JSON.parse ( text [ , reviver ] )
Before the parts that quote that section? It wasn't immediately obvious to me that they were from the section above.